zeq/hbp/detail/imageRawRGBA8.fbs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) 2015, Human Brain Project
+//                     Grigori Chevtchenko <grigori.chevtchenko@epfl.ch>
+
+namespace zeq.hbp;
+
+table ImageRawRGBA8
```

Comment:
We are discussing here actually what an Image class should be. RAW is a "format" in any image library and clearly separated. RGBA8 is just the type field in this Image class. 

Here we do not have a clear separation of classes, class hierarchies for images. We are overloading the class name for the fields and format and type is not present in the class, therefore it should at least be in the name.